doc/ref/gappkg.xml

```diff
@@ -73,10 +73,20 @@ be able to use some or all external binaries.
 
 <Index>automatic loading of GAP packages</Index>
 <Index>disable automatic loading</Index>
-Some &GAP; packages are prepared for <E>automatic loading</E>,
-that is they will be loaded automatically with &GAP;,
-others must in each case be separately loaded by a call to
-<Ref Func="LoadPackage"/>.
+Some &GAP; packages are loaded automatically with &GAP;. Those belong to
```

Comment:
Surely the average (or casual) GAP user just wants to know how to load a package?  So include here the material in the LoadPackage subsection which refers to this, and move the automatic loading stuff to a later subsection.

Reply:
I've reordered the text to mention `LoadPackage` at the start of the Section.